tst/CTA.WebForms2Blazor.Tests/FileConverters/ViewFileConverterTests.cs

```diff
@@ -30,47 +31,250 @@ public async Task HyperLinkControlConverter_Returns_Href_Node()
             string newPath = Path.Combine(FileConverterSetupFixture.TestFilesDirectoryPath, "HyperLinkControlOnly.razor");
             string relativePath = Path.GetRelativePath(FileConverterSetupFixture.TestProjectPath, newPath);
 
-            string expectedContents = @"
-<div class=""esh-pager"">
+            string expectedContents = @"<div class=""esh-pager"">
     <div class=""container"">
         <article class=""esh-pager-wrapper row"">
             <nav>
-                <a class=""esh-pager-item esh-pager-item--navigable="">
-                    Previous 
+                <a class='esh-pager-item esh-pager-item--navigable'>
+                    Previous
                 </a>
 
-                <span class=""esh-pager-item"">Showing <%: Model.ItemsPerPage%> of <%: Model.TotalItems%> products - Page <%: (Model.ActualPage + 1)%> - <%: Model.TotalPages%>
-                </span>
-
-                <a class=""esh-pager-item esh-pager-item--navigable="">
+                <a class='esh-pager-item esh-pager-item--navigable'>
                     Next
                 </a>
             </nav>
             <td>
-                <a href=""<%# GetRouteUrl(""EditProductRoute"", new {id =Item.Id}) %>"" class=""esh-table-link"">
+                <a href='@(GetRouteUrl(""EditProductRoute"", new {id =Item.Id}) )' class='esh-table-link'>
                     Edit
                 </a>
                 |
-                <a href=""<%# GetRouteUrl(""ProductDetailsRoute"", new {id =Item.Id}) %>"" class=""esh-table-link"">
+                <a href='@(GetRouteUrl(""ProductDetailsRoute"", new {id =Item.Id}) )' class='esh-table-link'>
                     Details
                 </a>
                 |
-                <a href=""<%# GetRouteUrl(""DeleteProductRoute"", new {id =Item.Id}) %>"" class=""esh-table-link"">
+                <a href='@(GetRouteUrl(""DeleteProductRoute"", new {id =Item.Id}) )' class='esh-table-link'>
                     Delete
                 </a>
             </td>
         </article>
     </div>
 </div>";
-            //This test below doesn't quite work yet, some issues with quotations that will be fixed in
-            // the next rules implementation PR
-            //Assert.AreEqual(expectedContents, fileContents);
+
+            Assert.AreEqual(expectedContents, fileContents);
```

Comment:
Is there a specific reason for single quotes? I believe double quotes are preferred in Blazor.

Reply:
I think using single quotes prevents it from turning into `&quote;` but if I can find a work around I'll use that instead